demo-app/src/main/java/com/gravatar/demoapp/MainActivity.kt

```diff
@@ -13,6 +13,7 @@ class MainActivity : ComponentActivity() {
         super.onCreate(savedInstanceState)
 
         // Initialize the Gravatar SDK with the API key if it is available
+        @Suppress("UNNECESSARY_SAFE_CALL")
```

Comment:
This is not related to this PR, is just to avoid the warning message as we want to keep the safe call.